packages/alchemy/src/client/modularAccountClient.ts

```diff
@@ -47,8 +46,7 @@ export function createModularAccountAlchemyClient<
     CustomTransport,
     Chain | undefined,
     MultiOwnerModularAccount<TSigner>,
-    BaseAlchemyActions<Chain | undefined, MultiOwnerModularAccount<TSigner>> &
```

Comment:
do we not need these `BaseAlchemyActions`?

Reply:
no not anymore, I fixed this cuz it was getting annoying to do this if you want to extend with other actions. The base client will always have these actions and anything else you pass in should be the extras you want to support